Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -11264,6 +11345,49 @@ - (void)dropInteraction:(UIDropInteraction *)interaction sessionDidEnd:(id<UIDro
 
 #endif // ENABLE(DRAG_SUPPORT)
 
+#pragma mark - Model Interaction Support
+#if ENABLE(MODEL_PROCESS)
+- (void)modelInteractionPanGestureDidBeginAtPoint:(CGPoint)inputPoint {
+    if (!_stageModeSession) {
+        _stageModeSession = { };
+        _page->requestInteractiveModelElementAtPoint(WebCore::roundedIntPoint(inputPoint));
+    }
+}
+
+// FIXME: https://bugs.webkit.org/show_bug.cgi?id=287601
+- (void)modelInteractionPanGestureDidUpdateWithPoint:(CGPoint)inputPoint {
+    if (_stageModeSession && !_stageModeSession.isPreparingForInteraction) {
```

Comment:
It's actually a bit unclear how this compiles, since `_stageModeSession` is a `std::optional<>`…